rpc/kate-rpc/src/lib.rs

```diff
@@ -54,6 +54,7 @@ where
 
 pub struct Kate<Client, Block: BlockT> {
 	client: Arc<Client>,
+	#[allow(clippy::type_complexity)]
```

Comment:
Consider using a `type` definition to simplify them. At least for `(Vec<...>, BlockDimensions)`.

Reply:
Fixed accordingly